tests/randoop/test/SequenceTests.java

```diff
@@ -141,6 +141,7 @@ private static void parseRecord(List<String> lines) {
     checkListsEqual(expectedCode, Arrays.asList(ds.toCodeString().split(Globals.lineSep)), testId);
   }
   
+  @SuppressWarnings("deprecation")
```

Comment:
Can you add a // comment on the same line indicating why this annotation is needed?
Should we update the code to eliminate the need for it?